front50-sql/src/main/kotlin/com/netflix/spinnaker/front50/model/SqlStorageService.kt

```diff
@@ -55,7 +56,8 @@ class SqlStorageService(
   private val jooq: DSLContext,
   private val clock: Clock,
   private val sqlRetryProperties: SqlRetryProperties,
-  private val chunkSize: Int
+  private val chunkSize: Int,
+  private val poolName: String = "default"
```

Comment:
I think if we define `poolName` via a configuration properties class, we can assign the default value there and pass it into the constructor and then remove the default value here.

Reply:
Took the same approach here, derive the default pool name from the existing connection pool settings.